src/workloads/execution/SecondaryReadsGenny.yml

```diff
@@ -0,0 +1,231 @@
+SchemaVersion: 2018-07-01
+Owner: "@TODO"  # what should go here? us or the owners of secondary_reads?
+Description: >
+ Test performance of secondary reads while applying oplogs
+
+ ### Setup
+
+ This workload runs only in a replica set with EXACTLY 3 NODES.
+ To use it in a replica set with a different number of nodes, adapt the
+ *number_of_nodes parameter (needs to be >1).
+
+ ### Test
+
+ On an empty replica set, the workload performs the following actions:
+ - Setup stage:
+    - Inserted *initial_nb_docs* documents into both the primary and the
+      secondaries. This should be done before any writes and reads.
+ - Actual test stage:
+    - *nWriters* background writers doing infinite writes on the primary which
+      will replicate these writes to secondaries.
+    - 1/16/32 readers doing queries on a sepcified seconary. Each query has
+      *batch_size* numbers of documents and only reads documents inserted in the
+      setup stage in order to have more constant behavior all the time.
```

Comment:
What constant behavior are we looking for here? Is this performance of the secondary read, so that we guarantee that the document we're trying to read already exists on the secondary?